examples/06-Define-your-own-architecture-with-Merlin-Models.ipynb

```diff
@@ -203,7 +203,7 @@
     }
```

Comment:
<p>may be write <code>ModelOutput</code> class? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/772/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>